### PR TITLE
Feature - Created route request unit tests

### PR DESCRIPTION
**Pull request by wickathou**
# Features
- [x] New tests for requests made for the Post and User controller

### CODE REVIEW

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  end
+
+  it 'check if user data is rendered' do
+    extra_user = User.create!(name: 'New Here', bio: 'With a bio')
+    get user_url(extra_user), as: :json
+    json_response = JSON.parse(response.body)
+    expect(json_response['user']).to eq(extra_user.as_json)
+    expect(json_response['posts']).to eq(extra_user.posts.as_json)
+  end
+end
```
Comment (Qoosim) [inline spec/requests/users_spec.rb]: - [x] Great job writing unit tests for the controllers in your app. However, there's one missing test that's yet to be written. Tests that ensures the response body includes the correct placeholder text. This should cover for both index and show pages for the user and the post controllers. It should look something similar to this :point_down:Kindly add that and you should be good to go. :+1:  

```
  expect(response.body).to include("Welcome to user index page 
   or whatever your placeholder test is")
```

### Project Requirements.
placeholder-test

- [Credit :+1: ] Extral credit is given to you for writing test for json format. You nailed it. :handshake:

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @wickathou, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  Implemented some project requirements. :heavy_check_mark: 
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/users_spec.rb (hunk 1 above)
Comment (peterrobert) [reply spec/requests/users_spec.rb]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @wickathou, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️
[ ] Great work adding the tests that were requested by the previous reviewer 👏🏿 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**